code/modules/mob/living/carbon/xenomorph/Castes/Larva.dm

```diff
@@ -25,7 +25,7 @@
 	evolves_to = list(/mob/living/carbon/Xenomorph/Drone, /mob/living/carbon/Xenomorph/Runner, /mob/living/carbon/Xenomorph/Sentinel, /mob/living/carbon/Xenomorph/Defender)
 
 	// *** Flags *** //
-	caste_flags = CASTE_CAN_BE_QUEEN_HEALED|CASTE_EVOLUTION_ALLOWED|CASTE_CAN_VENT_CRAWL|CASTE_INNATE_HEALING
+	caste_flags = CASTE_CAN_BE_QUEEN_HEALED|CASTE_EVOLUTION_ALLOWED|CASTE_CAN_VENT_CRAWL|CASTE_INNATE_HEALING|CASTE_NO_DECAY
```

Comment:
Can you avoid negative flags? That way we avoid double-negatives on checks. Something like `CASTE_ENERGY_PLASMA` or `CASTE_SILICON` would be better.

Reply:
Will `CASTE_DECAY_PROOF` suffice?

Reply:
Works.